changer-d-epoque.csl

```diff
@@ -0,0 +1,356 @@
+          <text variable="collection-number"/>
+        </group>
+      </group>
+    </macro>
+    <macro name="locator">
+      <group delimiter=" ">
+        <label variable="locator" form="short"/>
+        <text variable="locator"/>
+      </group>
+    </macro>
+    <macro name="locator-cit">
+      <choose>
+        <if type="article-journal article-magazine article-newspaper entry-dictionary entry-encyclopedia chapter paper-conference" match="any">
+          <group delimiter=" " prefix="en partic. ">
+            <label variable="locator" form="short"/>
+            <text variable="locator"/>
+          </group>
+        </if>
+        <else>
+          <text macro="locator"/>
+        </else>
+      </choose>
+    </macro>
+    <macro name="title">
+      <choose>
+        <if type="book" match="any">
+          <text variable="title" font-style="italic"/>
+        </if>
+        <else-if type="webpage" match="any">
+          <text variable="title"/>
+        </else-if>
+        <else-if variable="container-title" match="none">
+          <text variable="title" font-style="italic"/>
+        </else-if>
+        <else>
+          <text variable="title"/>
+        </else>
+      </choose>
+    </macro>
+    <macro name="publisher">
+      <group delimiter=", ">
+        <text variable="publisher-place"/>
+        <text variable="publisher"/>
+      </group>
+    </macro>
+    <macro name="year-date">
+      <choose>
+        <if variable="issued">
+          <date variable="issued">
+            <date-part name="year"/>
+          </date>
+        </if>
+        <else>
+          <text term="no date" form="short"/>
+        </else>
+      </choose>
+    </macro>
+    <macro name="edition">
+      <choose>
+        <if is-numeric="edition">
+          <group delimiter=" ">
+            <number variable="edition" form="ordinal"/>
+            <text term="edition" form="short" text-case="lowercase"/>
+          </group>
+        </if>
+        <else>
+          <text variable="edition"/>
+        </else>
+      </choose>
+    </macro>
+    <citation et-al-min="4" et-al-use-first="1" collapse="year">
+      <layout suffix="." delimiter=" ; ">
+        <choose>
+          <if position="ibid-with-locator">
+            <group delimiter=", ">
+              <text term="ibid" text-case="capitalize-first" font-style="italic"/>
+              <text macro="locator"/>
+            </group>
+          </if>
+          <else-if position="ibid">
+            <text term="ibid" text-case="capitalize-first" font-style="italic"/>
+          </else-if>
+          <else-if position="subsequent">
+            <group delimiter=", ">
+              <text macro="author"/>
+              <text variable="title" form="short" font-style="italic"/>
+              <group delimiter=" ">
+                <text term="cited" font-style="italic"/>
+                <text value="supra" font-style="italic"/>
+                <text term="note" form="short" suffix=" $$"/>
```

Comment:
In fact, I wanted not to hard code "n." as a recall for the first reference note, hence the term="note"
Testing it with the editor tool, it seems to work, it gives a text such as "op. cit. supra n. $$," the $$ being an on purpose reminder to give the first note where the work is cited.

If first-reference-note-number works with Zotero and Word it would be perfect. How is it supposed to be coded? It only appears in the specs under the variables, not anywhere else